src/test/java/org/broadinstitute/hellbender/tools/funcotator/FuncotatorDataSourceRetrieverIntegrationTest.java

```diff
@@ -0,0 +1,44 @@
+package org.broadinstitute.hellbender.tools.funcotator;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.testutils.ArgumentsBuilder;
+import org.testng.annotations.Test;
+
+/**
+ * Class to test the {@link FuncotatorDataSourceRetriever}.
+ * Created by jonn on 8/24/18.
+ */
+public class FuncotatorDataSourceRetrieverIntegrationTest extends CommandLineProgramTest {
+
+    //==================================================================================================================
+    // Private Static Members:
+
+    private static final boolean doDebugTests = false;
+
+    //==================================================================================================================
+    // Private Members:
+
+    //==================================================================================================================
+    // Helper Methods:
+
+    //==================================================================================================================
+    // Data Providers:
+
+    //==================================================================================================================
+    // Tests:
+
+    @Test(enabled = doDebugTests,
```

Comment:
How long does this test take to run?

Reply:
About an hour.  I disabled it because it's intractable to run it.